recipe/build.sh

```diff
@@ -11,7 +27,26 @@ FINAL_DEBUG_FFLAGS=FINAL_DEBUG_FFLAGS_${ctng_target_platform_u}
 FINAL_LDFLAGS=FINAL_LDFLAGS_${ctng_target_platform_u}
 FINAL_CONDA_PYTHON_SYSCONFIGDATA_NAME=FINAL_CONDA_PYTHON_SYSCONFIGDATA_NAME_${ctng_target_platform_u}
 
-if [ -z "${!FINAL_CFLAGS}" ]; then
+FINAL_CFLAGS="${!FINAL_CFLAGS}"
+FINAL_CXXFLAGS="${!FINAL_CXXFLAGS}"
+FINAL_FFLAGS="${!FINAL_FFLAGS}"
+FINAL_DEBUG_CFLAGS="${!FINAL_DEBUG_CFLAGS}"
+FINAL_DEBUG_CXXFLAGS="${!FINAL_DEBUG_CXXFLAGS}"
+FINAL_DEBUG_FFLAGS="${!FINAL_DEBUG_FFLAGS}"
+FINAL_LDFLAGS="${!FINAL_LDFLAGS}"
+FINAL_CONDA_PYTHON_SYSCONFIGDATA_NAME="${!FINAL_CONDA_PYTHON_SYSCONFIGDATA_NAME}"
+
+MAJOR_VERSION="${PKG_VERSION%%.*}"
+if [[ "${MAJOR_VERSION}" != 9 && "${MAJOR_VERSION}" != 10 ]]; then
+    FINAL_CXXFLAGS="$(echo $FINAL_CXXFLAGS | sed 's/-std=c++17 //g')"
+    FINAL_DEBUG_CXXFLAGS="$(echo $FINAL_DEBUG_CXXFLAGS | sed 's/-std=c++17 //g')"
+fi
+if [[ "${MAJOR_VERSION}" != 9 ]]; then
+    FINAL_FFLAGS="$(echo $FINAL_FFLAGS | sed 's/-fopenmp //g')"
+    FINAL_DEBUG_FFLAGS="$(echo $FINAL_DEBUG_FFLAGS | sed 's/-fopenmp //g')"
+fi
```

Comment:
A comment on why these flags are being excluded would be good. I suspect the change to -fopenmp is breaking for some folks, for example.